2.train-VAE/vae_utils.py

```diff
@@ -0,0 +1,115 @@
+from tensorflow.keras.layers import Lambda, Input, Dense, Dropout
```

Comment:
Add a citation to where you copied this code from (see comment above)

Reply:
this will also be removed for same reason as above